src/Microsoft.AspNetCore.Identity/SignInManager.cs

```diff
@@ -433,7 +433,19 @@ public virtual Task ForgetTwoFactorClientAsync()
         /// <param name="isPersistent">Flag indicating whether the sign-in cookie should persist after the browser is closed.</param>
         /// <returns>The task object representing the asynchronous operation containing the <see name="SignInResult"/>
         /// for the sign-in attempt.</returns>
-        public virtual async Task<SignInResult> ExternalLoginSignInAsync(string loginProvider, string providerKey, bool isPersistent)
+        public virtual Task<SignInResult> ExternalLoginSignInAsync(string loginProvider, string providerKey, bool isPersistent) 
```

Comment:
My suggestion is in following situation, only one method among all of its overloads should be virtual. The indention of the overloads seem to be fill in the default value. So this helps to avoid accidentally override the default value attempt in derived class.

None blocking. 

```
public virtual void Core(int parameter1, int parameter2)
{
    // body
}

public void Core(int parameter1)
{
    Core(parameter1, 0);
}

public void Core()
{
    Core(0, 0);
}
```